### DISCUSSION TITLE
replace text? is this possible currently

### DISCUSSION THREAD

**Discussion by MoprhiusX** [getgauge/taiko#2368] [General]
Hi team,
I been working with taiko for a bit now with some basic automation however I have come into a situation where I need to replace specific text in a text box. 

For example
Original: This is some awesome text
Need: This is some ok text

Basically replacing "awesome" with "ok" 

Is this something currently possible with taiko?

**Comment by zabil** [comment]
This can be done by [clearing](https://docs.taiko.dev/api/clear/) the current value and typing the new value for e.g.

```
await clear(textBox({placeholder:'Email'}))
await write("ok");
```

**Reply by MoprhiusX** [reply]
Hi Zabil,
Wont this clear the entire text box? Not just replace a specific word in the text boc

**Reply by zabil** [reply]
Yes that replaces the whole text. The best way to replace a part of the word is reading the original text and replacing the word using javascripts string [replace](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace), clearing the text box and the writing the new text.
